src/components v2/Molecules/Forms/InputSelect/InputSelect.spec.js

```diff
@@ -0,0 +1,73 @@
+import { render } from '@testing-library/vue';
+import InputSelect from './InputSelect.vue';
+
+describe('InputToggle.vue', () => {
```

Comment:
:eyes: is this the toggle?

Reply:
i going to choose to invoke my 5th amendment right